Vokal-Swift.xctemplate/BaseButton.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  ___FILENAME___
+//  ___PACKAGENAME___
+//
+//  Created by ___FULLUSERNAME___ on ___DATE___.
+//___COPYRIGHT___
+//
+
+import UIKit
+
+/**
+ * Base class for buttons. Extend this for any custom button classes, and override commonInit()
+ * to perform button setup there.
+ * Make sure to call super.commonInit() in your implementation!
+ */
+class BaseButton: UIButton, CommonInitializedView {
```

Comment:
Do you think it would be helpful to add either a `//MARK: -` in all these class where we are conforming to `CommonInitializedView`? Or maybe a separate extension?